src/main/java/org/mule/extension/file/common/api/matcher/PathMatcherPredicate.java

```diff
@@ -30,13 +29,29 @@ public final class PathMatcherPredicate implements Predicate<String> {
 
   private final Predicate<String> delegate;
 
+
+
   /**
    * Creates a new instance using the given pattern
    *
    * @param pattern the pattern to be used to test paths.
+   *
    */
+
   public PathMatcherPredicate(String pattern) {
-    delegate = getPredicateForFilename(pattern);
+    delegate = getPredicateForFilename(pattern, PredicateType.LOCAL_FILE_SYSTEM, true);
```

Comment:
Idem. I think the case-sensitive value should come from a constant that has the default value.